Makefile

```diff
@@ -48,16 +54,18 @@ pack-googet:
 	goopack -output_dir ./dist <(envsubst < ./.build/googet/google-cloudops-opentelemetry-collector.goospec)
 
 # tarball (Linux)
-
-# TODO
+.PHONY: linux-tarball
+linux-tarball:
+	make build
```

Comment:
I think it's convenient to separate the build and packaging commands. To be consistent with above can you make it:

```make
.PHONY: build-tarball
build-tarball:
	build package-tarball

.PHONY: package-tarball
package-tarball:
	./tar/generate_tar.sh
```
Also I think this works on OSs other than just Linux right? So probably drop the "(Linux)" comment above.

And also pls update the googet target names above so they are consistent (`build` & `package` instead of `package` & `pack`)

Reply:
updated. One remaining question thought. The syntax `build package-tarball` doesn't seem to work. I had to change this to `make build``make package-tarball` to have it working. Am I doing something wrong?

Reply:
Yea sorry makefiles are very sensitive. You can probably do:

```make
build-tarball: build package-tarball
```
or
```make
build-tarball:
	make build package-tarball
```
but not
```make
build-tarball:
	build package-tarball
```

Reply:
I was probably guilty of not testing that googet make command I checked in before 😮